### PR TITLE
kinder: add test workflows for 1.19

### PR DESCRIPTION
**Pull request by RA489**
xref https://github.com/kubernetes/kubeadm/issues/2076
test infra PR: https://github.com/kubernetes/test-infra/pull/17329

### CODE REVIEW

**Comment by neolit123** [comment]
@RA489 thanks for the PRs

we normally add these when the `release-1...` branch is created

https://github.com/kubernetes/kubeadm/issues/2076
> When the release-1.x branch is created

these extra jobs stress the google infrastructure (and this costs extra $$$), so we should not add them earlier. also currently master == 1.19.

but once the branch is created we can merge your PRs!

File: kinder/ci/workflows/external-etcd-1.19.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 1
+summary: |
+  This workflow tests the proper functioning of deploying an HA
+  cluster with secret copy using an external etcd cluster
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-external-etcd-1.19
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml
```
Comment (fabriziopandini) [inline kinder/ci/workflows/external-etcd-1.19.yaml:6]:
```suggestion
  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-etcd.yaml
```

File: kinder/ci/workflows/kustomize-1.19.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used for testing alternative
+  discovery methods for kubeadm join.
```
Comment (fabriziopandini) [inline kinder/ci/workflows/kustomize-1.19.yaml:4]:
```suggestion
  This workflow implements a sequence of tasks used for testing usage of kustomize
  patch with kubeadm.
```

File: kinder/ci/workflows/skew-master-on-1.19.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  This workflow tests the proper functioning of kubeadm version from master with Kubernetes v1.19
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-master-on-1.19
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-X-on-Y.yaml
+  config    > https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml
+vars:
+  kubeadmVersion: "{{ resolve `ci/latest` }}"
+  kubernetesVersion: "{{ resolve `ci/latest-1.19` }}"
+  controlPlaneNodes: 3
+tasks:
+- import: skew-x-on-y-tasks.yaml
```
Comment (fabriziopandini) [inline kinder/ci/workflows/skew-master-on-1.19.yaml:12]: please add CR

File: kinder/ci/workflows/external-etcd-1.19.yaml (hunk 1 above)
Comment (RA489) [reply kinder/ci/workflows/external-etcd-1.19.yaml]: Updated

File: kinder/ci/workflows/kustomize-1.19.yaml (hunk 2 above)
Comment (RA489) [reply kinder/ci/workflows/kustomize-1.19.yaml]: updated

File: kinder/ci/workflows/skew-master-on-1.19.yaml (hunk 3 above)
Comment (RA489) [reply kinder/ci/workflows/skew-master-on-1.19.yaml]: Updated

**Comment by RA489** [comment]
@neolit123 Looking for lgtm on this PR.

**Comment by neolit123** [comment]
thanks

/lgtm
/approve
/hold cancel

i've added comments to the test-infra PR.

**Pull request merged**